esx_service/utils/auth_api.py

```diff
@@ -190,7 +195,8 @@ def modify_privileges(privileges, allow_create, volume_maxsize_in_MB, volume_tot
     logging.debug("modify_privileges: allow_create=%s, volume_maxsize_in_MB=%s, volume_totalsize_in_MB=%s",
                   allow_create, volume_maxsize_in_MB, volume_totalsize_in_MB)
     if allow_create is not None:
```

Comment:
```if allow_create :``` means it is not None and also it is not boolean False. We have different actions for these; i.e. don't change the privilege if it is None and change the privilege to False if allow_create is False.